addon/templates/components/bs4/bs-accordion/item.hbs

```diff
@@ -0,0 +1,12 @@
+<div role="tab" {{action onClick value}} class="card-header {{if collapsed "collapsed" "expanded"}}">
```

Comment:
I think we should also move the common accordion item to the bs3 tree since it references the `panel` classes.